pkg/synopsysctl/cmd_create.go

```diff
@@ -578,13 +576,11 @@ var createPolarisCmd = &cobra.Command{
 			return err
 		}
 
-		myscheme := runtime.NewScheme()
-		_ = clientgoscheme.AddToScheme(myscheme)
-
-		// create controller runtime client
-		cl, _ := runtimeclient.New(restconfig, runtimeclient.Options{
-			Scheme: myscheme,
-		})
+		file, err := polarisToTmpComponentFile(*polarisObj)
```

Comment:
why need to be file?

Reply:
You're right, it doesn't need to be. I have updated the code to pass the component to kubectl using stdin.